challenges/2022-10-11-blast-from-the-past/setups/rust/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "rust"
+version = "0.1.0"
+edition = "2021"
```

Comment:
Ooooohhh! Not nightly? 😉

Reply:
No need to live on the "edge" for these, lol